app/helpers/sessions_helper.rb

```diff
@@ -42,4 +42,8 @@ def log_out
     reset_session
     @current_user = nil
   end
+
+  def admin?
+    current_user && current_user[:is_admin]
```

Comment:
```suggestion
    current_user&.is_admin
```
dc k. is_admin là 1 trường trong DB đúng k